src/Setup/Condition/PHPVersionCondition.php

```diff
@@ -27,7 +27,7 @@ public function __construct(string $which)
         return parent::__construct(
             "PHP version >= $which",
             fn (Setup\Environment $env) : bool => version_compare(phpversion(), $which, ">="),
-            "ILIAS 6 requires PHP $which or later."
+            "ILIAS 8 requires PHP $which or later."
```

Comment:
I think we should be able to use some `const` here, right?

Reply:
Definitely better for the future

Reply:
`ILIAS_VERSION_NUMERIC` could be used, but of course this is global scope :-). This could be either passed as a dependency or a new redundant constant could be introduced.

Reply:
This might be one of the few places where a global constant would actually be ok(ish).